Parse/ParseInstallation.cs

```diff
@@ -220,13 +220,13 @@ public ParseInstallation()
     internal override Task SaveAsync(Task toAwait, CancellationToken cancellationToken) {
       Task platformHookTask = null;
       if (CurrentInstallationController.IsCurrent(this)) {
-        SetIfDifferent("deviceType", ParseClient.PlatformHooks.DeviceType);
-        SetIfDifferent("timeZone", ParseClient.PlatformHooks.DeviceTimeZone);
-        SetIfDifferent("localeIdentifier", GetLocaleIdentifier());
-        SetIfDifferent("parseVersion", GetParseVersion().ToString());
-        SetIfDifferent("appVersion", ParseClient.PlatformHooks.AppBuildVersion);
-        SetIfDifferent("appIdentifier", ParseClient.PlatformHooks.AppIdentifier);
-        SetIfDifferent("appName", ParseClient.PlatformHooks.AppName);
+        ForceSetIfDifferent("deviceType", ParseClient.PlatformHooks.DeviceType);
```

Comment:
It does check for mutability when it calls `Remove`, though.I guess we can change each of these `SetIfDifferentCalls` and bound them by a null check on the value, although I'm not certain that makes a ton of sense (we could end up with really stale values).